envoy/upstream/upstream.h

```diff
@@ -543,6 +543,7 @@ class PrioritySet {
   COUNTER(upstream_cx_connect_attempts_exceeded)                                                   \
   COUNTER(upstream_cx_connect_fail)                                                                \
   COUNTER(upstream_cx_connect_timeout)                                                             \
+  COUNTER(upstream_cx_connect_with_0_rtt)                                                          \
```

Comment:
This stat should be documented somewhere. Can you check where the others are documented?

Reply:
done